tests/vars.sample.yaml

```diff
@@ -49,3 +49,6 @@ ironic_adoption: false
 
 # Run pre-adoption validation before the deploying
 run_pre_adoption_validation: true
+
+# Whether the adopted node will host compute services
+compute_adoption: true
```

Comment:
what are the cases when we have that set to False?

Reply:
@bogdando one can set it to `false` and validate that `test-minimal` target still passes without deploying `libvirt` and `nova-compute`. Does it address your concern?